apps/idos-data-dashboard/src/routes/dashboard/wallets/components/wallet-card.tsx

```diff
@@ -19,14 +19,25 @@ export const WalletCard = ({ address, isDisabled, onDelete }: WalletCardProps) =
           </Text>
         </VStack>
       </HStack>
-      <Button
-        isDisabled={isDisabled}
-        id={`delete-wallet-${address}`}
-        leftIcon={<XIcon size={20} />}
-        onClick={() => onDelete(address)}
+      <Tooltip
+        hasArrow
+        bg="neutral.500"
+        px={2}
+        py={0.5}
+        rounded="md"
+        isDisabled={!isDisabled}
```

Comment:
wat :|

This line is going to confuse a lot of future readers. :sweat_smile:

Reply:
Ah, I finally got it. Can you rename `WalletCard`'s `isDisabled` param to `canDelete`? Doesn't feel like a big change to do now, and it'll avoid this weird phrasing.